usage.md

```diff
@@ -0,0 +1,46 @@
+
+<h1><img align="center" height="50" src="logo.png"> nbgitpuller extension</h3>
```

Comment:
added file. Note: images like the logo are outdated, as the new logo is part of a new rebranding branch, and will work once merged.